OlinkAnalyze/DESCRIPTION

```diff
@@ -41,8 +41,10 @@ Authors@R: c(
            comment = c(Github = "olofmansson")),
     person("Ola", "Caster", role = "ctb",
            comment = c(Github = "OlaCaster")),
-    person("Kristyn", "Chin", role = "ctb",
+    person("Kristyn", "Chin", role = "aut",
```

Comment:
Only potential change - moving up below other authors for consistency. Not required though.